app/views/news/show.html.erb

```diff
@@ -27,24 +27,31 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
-<% content_for :action_menu_specific do %>
-  <%= link_to(l(:button_edit),
-        edit_news_path(@news),
-        :class => 'icon icon-edit',
-        :accesskey => accesskey(:edit),
-        :onclick => 'Element.show("edit-news"); return false;') if User.current.allowed_to?(:manage_news, @project) %>
-  <%= watcher_link(@news, User.current) %>
-  <%= link_to(l(:button_delete),
-        news_path(@news),
-        :confirm => l(:text_are_you_sure),
-        :method => :delete,
-        :class => 'icon icon-delete') if User.current.allowed_to?(:manage_news, @project) %>
+<%= toolbar title: "#{avatar(@news.author)} #{@news.title}" do %>
```

Comment:
Ignoring the XSS part, the current implementation will still suffer from a bad title text.

I'd probably not call the thing `avatar`, but look for something more generic, e.g. `thumbnail`, because I could imagine it being used for other images than avatars as well.

I also agree that deleting the avatar is probably not the best course of action.